src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -64,140 +64,137 @@ const defaultProps = {
     reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
 };
 
-class WorkspaceReimburseView extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            currentRatePerUnit: this.getCurrentRatePerUnitLabel(),
-        };
-    }
-
-    componentDidMount() {
-        this.fetchData();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (prevProps.policy.customUnits !== this.props.policy.customUnits || prevProps.preferredLocale !== this.props.preferredLocale) {
-            this.setState({currentRatePerUnit: this.getCurrentRatePerUnitLabel()});
-        }
-
-        const reconnecting = prevProps.network.isOffline && !this.props.network.isOffline;
-        if (!reconnecting) {
-            return;
-        }
-
-        this.fetchData();
-    }
-
-    getCurrentRatePerUnitLabel() {
-        const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), (unit) => unit.name === 'Distance');
-        const customUnitRate = _.find(lodashGet(distanceCustomUnit, 'rates', {}), (rate) => rate.name === 'Default Rate');
-        const currentUnit = this.getUnitLabel(lodashGet(distanceCustomUnit, 'attributes.unit', 'mi'));
-        const currentRate = this.getRateLabel(customUnitRate);
-        const perWord = this.props.translate('common.per');
+function WorkspaceReimburseView(props) {
+    const [currentRatePerUnit, setCurrentRatePerUnit] = useState('');
+    const viewAllReceiptsUrl = `expenses?policyIDList=${props.policy.id}&billableReimbursable=reimbursable&submitterEmail=%2B%2B`;
+    const distanceCustomUnit = _.find(lodashGet(props.policy, 'customUnits', {}), (unit) => unit.name === 'Distance');
+    const distanceCustomRate = _.find(lodashGet(props.distanceCustomUnit, 'rates', {}), (rate) => rate.name === 'Default Rate');
```

Comment:
NAB, Why `props.distanceCustomUnit`. Looks like previously it was derived from `distanceCustomUnit` constant

Reply:
yes it should have been taken from the variable. I've fixed it.

Reply:
Whoops, my bad missed that spot 😅